.NET/Microsoft.Recognizers.Text.DateTime/English/Parsers/EnglishMergedParserConfiguration.cs

```diff
@@ -11,7 +11,8 @@ public EnglishMergedParserConfiguration(IDateTimeOptionsConfiguration config)
         {
             BeforeRegex = EnglishMergedExtractorConfiguration.BeforeRegex;
             AfterRegex = EnglishMergedExtractorConfiguration.AfterRegex;
-            SinceRegex = EnglishMergedExtractorConfiguration.SinceRegex;
+            SinceRegex = (config.Options & DateTimeOptions.ExperimentalMode) != 0 ? EnglishMergedExtractorConfiguration.SinceRegexExp :

```

Comment:
Do we need this logic here if it's been properly assigned in EnglishMergedExtractorConfiguration.cs?

Reply:
yes, as commented in #2230 this seems to be necessary. Otherwise, the regex in MergedParser always gets the standard expression even in Experimental mode.

Reply:
I do not know if there is a different way to do the assignment in the Extractor configuration file which allows to avoid this logic.

Reply:
Let's see. For now let's use this version.